src/substrate/filters/enricher.ts

```diff
@@ -714,6 +714,25 @@ export async function Enrich(
     excludeAddresses?: string[],
   }> => {
     switch (kind) {
+      case EventKind.DemocracyVoted: {
+        const voter = extrinsic.signer.toString();
+        const [ idx, vote ] = extrinsic.args as [ Compact<ReferendumIndex>, AccountVote ];
+        if (vote.isSplit) {
+          throw new Error('split votes not supported');
```

Comment:
Will throwing an error break anything?

Reply:
The pattern in this file is to throw and handle outside of it, so no.